C/CLIME/build_tarballs.jl

```diff
@@ -22,7 +22,7 @@ make install
 
 # These are the platforms we will build for by default, unless further
 # platforms are passed in on the command line
-platforms = supported_platforms()
+platforms = supported_platforms(;experimental=true)
```

Comment:
Uhm, in order to change julia compat you must change the version number: #2763.  It's ok for AlgRemez, but not this package

Reply:
Thanks. I changed the version number of the CLIME.